lib/stellar_core_commander/local_process.rb

```diff
@@ -166,6 +166,7 @@ def config
         INVARIANT_CHECKS=#{invariants.to_s}
 
         [QUORUM_SET]
+        THRESHOLD_PERCENT=51
```

Comment:
why do we need this smaller number than the default? It it that in other places we use 51 as threshold?

Reply:
yes, used in `docker_process.rb`